test/data/html/tests.js

```diff
@@ -761,8 +761,24 @@ exports.test_data = {
             { fragment: true, unchanged: '<div class="searchform"><input type="text" value="" name="s" id="s"><input type="submit" id="searchsubmit" value="Search"></div>' },
         ]
     }, {
+        name: "unformatted tags with namespaces",
+        description: "namespaced tags with colons can be specified for leaving unformatted",
+        options: [ {name: 'unformatted', value: '[ "name:space"]'}],
+        tests: [{
```

Comment:
Please add more tests, including some tests that show invalid elements such as `a:b:c`
Also, need to support and test these elements when they are not unformatted.